src/com/itschool/session4/homework/binaryarithmeticoperators/ex1.java

```diff
@@ -0,0 +1,38 @@
+package com.itschool.session4.homework.binaryarithmeticoperators;
+
+import java.util.Scanner;
+
+/*Write a Java program to take two numbers as input and display the result of addition,
+subtraction, multiplication, division, and modulus.
+ */
+
+
+public class ex1 {
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Enter the first number: ");
+        double nr1;
+        nr1 = scanner.nextDouble();
+
+        System.out.print("Enter the second number: ");
+        double nr2;
+        nr2 = scanner.nextDouble();
+
+        double addition = nr1 + nr2;
+        double subtraction = nr1 - nr2;
+        double multiplication = nr1 * nr2;
+        double division = nr1 / nr2;
+        double modulus = nr1 % nr2;
+
+        System.out.println("Addition: " + addition);
+        System.out.println("Subtraction: " + subtraction);
+        System.out.println("Multiplication: " + multiplication);
+        System.out.printf("Division: %.2f\n" , division);
+        System.out.println("Modulus: " + modulus);
+
+        scanner.close();

```

Comment:
can you explain what this line does and what is the purpose of it?

Reply:
%.2f syntax tells Java to return the variable (division) with 2 decimal places and added  "\n" next to it to display it on a separate line, %.2f can also be used with println by combining it with String.format 
System.out.println("Division: " + String.format("%.2f" , division)); 
The scanner.close() method is used to close the Scanner object in Java.

Reply:
Ok, but I was curious why closing this scanner/how does it help?
Closing resources is the right way (most of the times also the only way) for the things working correctly, but in this case is not mandatory since the scanner wraps the builtin jdk input system. Moreover your program exits after finishing the code from this class, therefore all resources are being destroyed.
In short, in this situation does not make any difference. I just want to check why you wrote this line :p